libs/components/pages/src/lib/modules/action-hub/types/page-link.ts

```diff
@@ -1,9 +1,11 @@
 import { NavigationExtras } from '@angular/router';
 
+import { SkyPageLinkInterface } from './page-link-interface';
+
 /**
  * Displays links to related information or recently accessed items.
  */
-export interface SkyPageLink {
+export interface SkyPageLink extends SkyPageLinkInterface {
   /**
    * Specifies the link text.
    */
```

Comment:
I don't believe we need to have `label` in this interface any more as it is a part of `SkyPageLinkInterface`

Reply:
Good catch.